docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml

```
@@ -1124,7 +1124,12 @@ methods:
     uid: 'ExcelScript!ExcelScript.Range#getImage:member(1)'
     package: ExcelScript!
     fullName: getImage()
-    summary: Renders the range as a base64-encoded png image.
+    summary: >-
+      Renders the range as a base64-encoded png image.
+
+
+      **Note**: There is a known issue with `Range.getImage` that causes longer or wrapped text string to render without
+      line wrapping and in smaller cells. This makes the resulting image unreadable.
```

Two parts: 
1. Based on the customer photo, it looks like this issue also affects surrounding cells – not just the cell containing the wrapped text. 
2. I think the description should offer guidance or a workaround. If there's no workaround, then perhaps something like: 
```suggestion
      line wrapping and in smaller cells. The wrapped text may merge with nearby cells, making the resulting image unreadable. Avoid using `Range.getImage` for strings longer than ___ characters and for cells with text wrapping enabled.
```